Off-Top-App/off-top-flutter
# OT-CreateReportListView | Created a Reports table to show the users data in the reports page

**What I did:**

- I created a scrollable  table in the reports page to show user reports.

**How to test:**

- Start the zookeeper server.

- Start the kafka server.

- Start Java spring-boot.

- Make sure port forwarding is active.

- Start up the OFF-TOP app.

- Go to the Reports page.

- The Reports table should be shown along with the user reports and the table should be vertically scrollable.

![Capture-1](https://user-images.githubusercontent.com/56651529/85535598-30cbbf80-b5c7-11ea-8832-3ae0013d4467.PNG)
